resolwe_bio/processes/workflows/chip_seq.yml

```diff
@@ -357,3 +357,341 @@
+            using all regions not too low (>10) and not too high (<30) to build paired-peaks
+            model. If MACS can not find more than 100 regions to build model, it will use the
+            --extsize parameter to continue the peak detection ONLY if --fix-bimodal is set.
+        - name: mfold_upper
+          label: MFOLD range (upper limit)
+          type: basic:integer
+          required: false
+          description: |
+            This parameter is used to select the regions within MFOLD range of high-confidence
+            enrichment ratio against background to build model. The regions must be lower than
+            upper limit, and higher than the lower limit of fold enrichment. DEFAULT:10,30 means
+            using all regions not too low (>10) and not too high (<30) to build paired-peaks
+            model. If MACS can not find more than 100 regions to build model, it will use the
+            --extsize parameter to continue the peak detection ONLY if --fix-bimodal is set.
+        - name: slocal
+          label: Small local region
+          type: basic:integer
+          required: false
+          description: |
+            Slocal and llocal parameters control which two levels of regions will be checked
+            around the peak regions to calculate the maximum lambda as local lambda. By default,
+            MACS considers 1000bp for small local region (--slocal), and 10000bps for large local
+            region (--llocal) which captures the bias from a long range effect like an open
+            chromatin domain. You can tweak these according to your project. Remember that if the
+            region is set too small, a sharp spike in the input data may kill the significant
+            peak.
+        - name: llocal
+          label: Large local region
+          type: basic:integer
+          required: false
+          description: |
+            Slocal and llocal parameters control which two levels of regions will be checked
+            around the peak regions to calculate the maximum lambda as local lambda. By default,
+            MACS considers 1000bp for small local region (--slocal), and 10000bps for large local
+            region (--llocal) which captures the bias from a long range effect like an open
+            chromatin domain. You can tweak these according to your project. Remember that if the
+            region is set too small, a sharp spike in the input data may kill the significant
+            peak.
+        - name: extsize
+          label: extsize
+          type: basic:integer
+          required: false
+          description: |
+            While '--nomodel' is set, MACS uses this parameter to extend reads in 5'->3' direction
+            to fix-sized fragments. For example, if the size of binding region for your
+            transcription factor is 200 bp, and you want to bypass the model building by MACS,
```

Comment:
Sometimes bp is written without a space (i.e. 200bp) and with space (200 bp). Is this something you feel we need to standardize?

Reply:
Using a space is the correct way of writing units. Fixed in the related processes.